gradle/testing/randomization/policies/tests.policy

```diff
@@ -64,6 +64,9 @@ grant {
   permission java.lang.RuntimePermission "getClassLoader";
   permission java.lang.RuntimePermission "setContextClassLoader";
 
+  // TestLockFactoriesMultiJVM opens a random port on 127.0.0.1 (port 0 = ephemeral port range):
```

Comment:
This property wouldn't be needed if you forked the "server: side as well as clients... :)

Reply:
This does not work, as there needs to be a communication, so the test known when to start the clients. This was not different in Ant (the main process was running fork=false, the clients fork=true)